MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -538,14 +542,54 @@ public string GetTextSubtitleParam(EncodingJobInfo state)
                 ? string.Empty
                 : string.Format(",setpts=PTS -{0}/TB", seconds.ToString(_usCulture));
 
-            string fallbackFontParam = string.Empty;
+            // TODO
```

Comment:
This I can explain : I did the same with the first port. No IApplicationPaths is available in the place where the constructor is called, so I commented out instead of removing, because this is probably fixable by getting the program data path or the assemblyinfo from elsewhere, but I couldn't find out easily how on my first port.